packages/kbn-test-subj-selector/index.js

```diff
@@ -19,12 +19,20 @@
 
 function selectorToTerms(selector) {
   return selector
-    .replace(/\s*&\s*/g, '&') // remove all whitespace around joins
-    .split(/\s+/);
+    .replace(/\s*~\s*/g, '~') // css locator with '~' operator cannot contain spaces
+    .replace(/\s*>\s*/g, '>') // remove all whitespace around joins >
+    .replace(/\s*&\s*/g, '&') // remove all whitespace around joins &
+    .split(/>+/);
 }
 
 function termToCssSelector(term) {
-  return term ? '[data-test-subj~="' + term + '"]' : '';
+  if (term) {
+    return term.startsWith('~')
+      ? '[data-test-subj~="' + term.substring(1).replace(/\s/g, '') + '"]'
```

Comment:
What's the `.replace()` here for? Doing something like `~foo bar`  shouldn't make `[data-test-subj~=foobar]` it should probably be `[...~=foo bar]`.

Reply:
@spalger If got it right, `[...~=foo bar]` won't work per design, since CSS selector does not support a combination of `contains` and value with spaces:
![exa](https://user-images.githubusercontent.com/10977896/64652419-1a7fb900-d424-11e9-90a5-2035cee675f9.png)

`.replace(/\s/g, '')` is used to remove spaces and make locator valid, but maybe it is better to throw Error stating `~` and value with spaces cannot be used (suggesting to use ~value1 & ~value2)?